search/search-domain/src/main/java/io/camunda/search/filter/MappingFilter.java

```diff
@@ -7,13 +7,18 @@
  */
 package io.camunda.search.filter;
 
+import static io.camunda.util.CollectionUtil.addValuesToList;
+
 import io.camunda.util.ObjectBuilder;
+import java.util.List;
 
-public record MappingFilter(Long mappingKey, String claimName, String claimValue)
+public record MappingFilter(
+    Long mappingKey, String claimName, List<String> claimNames, String claimValue)
```

Comment:
Can we replace the singular `claimName` filter with a single `claimNames` list filter? The API would be a bit simpler if there's only one way to express a filter.

Reply:
We could but for now I'd prefer to keep it this way, its a pattern that we use consistently in other filters where there could be one or multiple keys i.e. AuthorizationFilter.